tests/ETHTornado.test.ts

```diff
@@ -99,11 +116,13 @@ describe("ETHTornado", function () {
             tornado.filters.Deposit(),
             receipt.blockHash
         );
+        // XXX Tests fail here
+        // AssertionError: expected '0xf3c07b10bdcef09c269e633587caa3885fb204ddbfee021fdb8810398a5b2707' to equal '0x2b0f6fc0179fa65b6f73627c0e1e84c7374d2eaec44c9a48f2571393ea77bcbb'
```

Comment:
I'm getting a different error on my local test. 3 tests all fail with the same error message.

```
  1) ETHTornado
       deposit and withdraw:
     Error: VM Exception while processing transaction: reverted with reason string '_left should be inside the field'
```

The value '0xf3c07b...' looks not like a valid value from the poseidon hash, since this value is greater than the bn field size '21888...'.
```python
>>> 0xf3c07b10bdcef09c269e633587caa3885fb204ddbfee021fdb8810398a5b2707 > 21888242871839275222246405745257275088548364400416034343698204186575808495617
True
```